packages/flat-components/src/components/CloudStorage/CloudStorageFileTitle/index.tsx

```diff
@@ -26,12 +30,15 @@ export interface CloudStorageFileTitleProps
     convertStatus?: FileConvertStep;
     /** Is title clickable. Default false */
     titleClickable?: boolean;
-    /** When title is clicked */
-    onTitleClick?: (fileUUID: string) => void;
     /** UUID of file that is under renaming */
     renamingFileUUID?: string;
+    resourceType?: ResourceType;
+    parentDirectoryPath?: string;
+    /** When title is clicked */
+    onTitleClick?: (fileUUID: string, pushHistory: (path: string) => void) => void;
     /** Rename file. Empty name for cancelling */
     onRename?: (fileUUID: string, fileName?: CloudStorageFileName) => void;
+    onNewDirectoryFile?: (directoryInfo: DirectoryInfo) => Promise<void>;
```

Comment:
```suggestion
    renamingFileUUID?: string;
    resourceType?: ResourceType;
    parentDirectoryPath?: string;
    onTitleClick?: (fileUUID: string, pushHistory: (path: string) => void) => void;
    onRename?: (fileUUID: string, fileName?: CloudStorageFileName) => void;
    onNewDirectoryFile?: (directoryInfo: DirectoryInfo) => Promise<void>;
```

Let's remove those unused comments.

Ditto for all other props.